server/app/org/scalaexercises/exercises/services/ExercisesService.scala

```diff
@@ -136,11 +140,25 @@ sealed trait RuntimeSharedConversions {
       name = library.name,
       description = library.description,
       color = library.color getOrElse "black",
+      logoPath = library.logoPath,
+      logoData = loadLogoSrc(library.logoPath),
       sections = library.sections map convertSection,
       timestamp = library.timestamp,
       buildInfo = convertBuildInfo(library.buildMetaInfo)
     )
 
+  def loadLogoSrc(logoPath: String): Option[String] = {
+    def checkLogoPath(): Boolean =
+      Option(getClass.getClassLoader.getResource(logoPath + ".svg")).isDefined
+
+    def logoData(path: String): Option[String] =
+      Option(getClass().getClassLoader.getResourceAsStream(path))
+        .map(stream ⇒ new BASE64Encoder().encode(IOUtils.toByteArray(stream)))
```

Comment:
Maybe we could create a single instance of `BASE64Encoder` in this `ExercisesService` object that we could re-use each time.